ern-container-gen-ios/src/hull/ElectrodeContainer/ElectrodeReactNative.m

```diff
@@ -381,7 +381,7 @@ - (void)addExtraDevMenu {
     RCTDevMenuItem *enableMinifyMenu = [RCTDevMenuItem buttonItemWithTitle:minifyTitle handler:^{
         [[RCTBundleURLProvider sharedSettings] setEnableMinification:!enableMinify];
         __strong RCTBridge *strongBridge = self.bridge;
-        strongBridge.bundleURL = [[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:@"index" fallbackResource:nil];
+        strongBridge.bundleURL = [[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:@"index"];
```

Comment:
Is this change related to the new architecture?

Reply:
not related to new arch. But again from changelog from 68.0 had breaking changes.